src/server/pps/server/s3g_sidecar.go

```diff
@@ -309,18 +266,17 @@ func (s *k8sServiceCreatingJobHandler) OnCreate(ctx context.Context, jobInfo *pp
 		return nil
 	})
 	if err != nil {
-		logrus.Errorf("could not create service for job %q: %v", jobInfo.Job.ID, err)
+		logrus.Errorf("could not create service for job %q: %v", jobInfo.Job.String(), err)
```

Comment:
you don't actually have to call String() here, `Printf` etc will do that automatically

Reply:
that's right! good catch